pkg/balancer-interfaces/contracts/solidity-utils/helpers/IAuthorizerAdaptor.sol

```diff
@@ -14,8 +14,9 @@
 
 pragma solidity ^0.7.0;
 
-import "../solidity-utils/helpers/IAuthentication.sol";
-import "../vault/IVault.sol";
+import "../../vault/IVault.sol";
+
+import "./IAuthentication.sol";
 
 interface IAuthorizerAdaptor is IAuthentication {
```

Comment:
If moving this file, it should go to the 'standalone-utils' directory, not 'solidity-utils'.

Reply:
Leaving it, consistent with other changes